tools/rtpsrelay/rtps.ini

```diff
@@ -16,6 +16,6 @@ DiscoveryConfig=application_rtps_discovery
 
 [rtps_discovery/application_rtps_discovery]
 SedpMulticast=0
-SpdpRtpsRelayAddress=127.0.0.1:4444
-SedpRtpsRelayAddress=127.0.0.1:4445
-#RtpsRelayOnly=1
+SpdpRtpsRelayAddress=localhost:4444
+SedpRtpsRelayAddress=localhost:4445
+RtpsRelayOnly=1
```

Comment:
Is this change meant to be here?  (Also with 127.0.0.1 changing to localhost -- not sure if we want it)